geometry_msgs/PointStamped Target
string Color
string Status
